contracts/javascore/example-dapps/dapp-multi-protocol/src/main/java/xcall/sample/dapp/MultiProtocolSampleDapp.java

```diff
@@ -131,7 +131,11 @@ public void handleCallMessage(String _from, byte[] _data, String[] protocols) {
             Context.require(equals(protocols, getSources(from.net())), "invalid protocols");
 
             Context.require(!new String(_data).equals("rollback"), "failed");
-            // normal message delivery
+
+            if (new String(_data).equals("reply-response")) {
+                // response message
+                _sendCallMessage(Context.getValue(), _from, new byte[] { 1, 2, 3 }, null);
```

Comment:
User BigInteger.ZERO instead of Context.getValue. Just so it clear that it is supposed to be free